client/src/components/BackgroundPanel.tsx

```diff
@@ -29,6 +31,13 @@ export interface IResponseData {
 	id: string;
 }
 
+export interface IResponseData2 {
+	url: string;
+	origin: {
+		title: string;
+	};
+}
+
```

Comment:
Poor naming convention, if there's more than 1 response data each should further indicate what type of response, like IResponseDataGoogle